src/dbt_client/dbtCoreIntegration.ts

```diff
@@ -332,14 +332,19 @@ export class DBTCoreProjectIntegration
             python!`to_dict(project.execute_macro('get_show_sql', ${args}, ${query}))`,
         );
 
+        // `get_show_sql` adds extra newlines at the start of the query
```

Comment:
@mdesmet @anandgupta42 since we are showing line numbers in sql tab in query results panel, we need to add this trim logic to make sure the sql code line numbers match with the sql in editor. If this is not needed (ie, if line numbers does not need to match with editor), we can remove this.

Reply:
I don't think that is required. Compiled sql and jinja sql does not need to match